### PR TITLE
fix: lost state of the children when the tooltip disparears

### PR DESCRIPTION
**Pull request by jare-talend**
**What is the problem this PR is trying to solve?**
When the tooltip disappears, the children come back to the previous state.
To reproduce the bug: http://talend.surge.sh/components/?selectedKind=ActionDropdown&selectedStory=default&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel

On the last dropdown, click on the dropdown to expand it. Move the mouse. The toolip disappears and the dropdown collapse it.

**What is the chosen solution to this problem?**

Don't remove the TooltipTrigger after his creating.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Comment by acateland** [comment]
DON'T MERGE THIS AS IT REQUIRE DISCUSSION
see https://github.com/Talend/ui/pull/478

**Comment by jare-talend** [comment]
Except this bug is very blocking for us.

File: packages/components/src/TooltipTrigger/TooltipTrigger.component.js [hunk 1]
```diff
@@ -78,7 +78,6 @@ class TooltipTrigger extends React.Component {
 				placement={this.props.tooltipPlacement}
 				overlay={tooltip}
 				delayShow={400}
-				onExited={this.onMouseOut}
```
Comment (acateland) [inline packages/components/src/TooltipTrigger/TooltipTrigger.component.js]: the bind of this.onMouseOut id removed but the function attached to the component remain and is unused.

Comment (jare-talend) [inline packages/components/src/TooltipTrigger/TooltipTrigger.component.js]: exact

**Pull request merged**